frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.jsx

```diff
@@ -184,7 +184,11 @@ class FieldValuesWidgetInner extends Component {
   fetchFieldValues = async query => {
     if (query == null) {
       const { fields, fetchFieldValues } = this.props;
-      await Promise.all(fields.map(field => fetchFieldValues(field.id)));
+      await Promise.all(
+        fields
+          .filter(field => !field.isVirtual())
```

Comment:
:x: 

I tried the branch locally and while reproducing it (just followed the steps) I noticed that there's a JS error coming from `dedupeValues` call right in the next line (192). It also happens afterwards, when trying to edit the filter.
The error is caused by this change.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '0')
    at FieldValuesWidget.jsx:458:1
    at Array.map (<anonymous>)
    at dedupeValues (FieldValuesWidget.jsx:458:1)
    at _callee$ (FieldValuesWidget.jsx:184:1)
    at tryCatch (runtime.js:63:1)
    at Generator.invoke [as _invoke] (runtime.js:293:1)
    at Generator.next (runtime.js:118:1)
    at asyncGeneratorStep (index.jsx:1:1)
    at _next (index.jsx:1:1)
```

![image](https://github.com/metabase/metabase/assets/6830683/eefade83-755d-4652-8190-547e7dab3e4a)